tutorials/3d/physical_light_and_camera_units.rst

```diff
@@ -0,0 +1,260 @@
+.. _doc_physical_light_and_camera_units:
+
+Physical light and camera units
+===============================
+
+Why use physical light and camera units?
+----------------------------------------
+
+Since video game graphics generally don't aim for realism, Godot uses arbitrary
+units by default to favor usability. This applies to light color and energy,
+camera field of view, exposure, and more.
```

Comment:
```suggestion
Godot uses arbitrary units for many physical properties that apply to light
like color, energy, camera field of view, and exposure. By default these
properties use arbitrary units because using accurate physical units comes
with a few tradeoffs that are not worth making for many games. As Godot
favors ease-of-use by default, physical light units are disabled by default.
```